Library/Tracking/KSRAnalytics.swift

```diff
@@ -1389,6 +1389,22 @@ public final class KSRAnalytics {
     )
   }
 
+  /**
+   Call when a user clicks creator's name on a project.
+
+   - parameter project: The project the creator's name is clicked from.
+   */
+
+  public func trackGotoCreatorDetailsClicked(project: Project) {
+    let props = projectProperties(from: project, loggedInUser: self.loggedInUser)
+      .withAllValuesFrom(contextProperties(ctaContext: .creatorDetails, page: .projectPage))
```

Comment:
I don't see a `context_page` defined in the ticket?

Reply:
Well, there was another ticket that doesn't have a context_page as well, I asked Laurie and she said context_page should always be page the event was triggered from.

Reply:
Ok can you leave a comment on the ticket indicating you are adding this context_page property so it is documented somewhere. Tag Laurie for visibility.